en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc

```diff
@@ -230,6 +230,65 @@ When there are null values in custom expressions and are:
 * Fields without attributes, the null value is ignored. If the value in a required field, an error message occurs. If the value is in an optional field, the value is ignored and not sent to the connector. +
 * Fields with attributes, the value is ignored and not sent to the connector.
 
+=== Create a Picklist Mapping
+
+Using Picklist Mapping, you can map data from one application to the corresponding data needed by another application and then reuse those mappings in different flows in your organization. This allows you to connect two applications that have different values for similar fields and to align the values in those fields across applications. This enables you to determine how the possible values of an output field (data pill) map to the possible values of an input field.
+
+For example, you can map a data pill from the *Country* field in a Salesforce record and provide the same values in the *Country* field of the NetSuite record.
+
+[%header%autowidth.spread]
+|===
+|Country (Salesforce) | Country (NetSuite)
+|US |_unitedStates
+|CA |_canada
+|DE |_germany
+|AR |_argentina
+|===
+
+In this example, the *Status* field in a Case in Salesforce is mapped to the *Status* field in a Ticket in Zendesk to create continuity in field values across systems.
```

Comment:
should Case be capitalized? Should it be the *Status* field in a Salesforce case?

Reply:
Ticket shouldn't be capitalized right?